be/src/runtime/runtime_state.h

```diff
@@ -352,6 +352,8 @@ class RuntimeState {
         return _query_options.__isset.skip_delete_bitmap && _query_options.skip_delete_bitmap;
     }
 
+    bool enable_page_cache() const { return _query_options.enable_page_cache; }
```

Comment:
```cpp
return _query_options.__isset.enable_page_cache && _query_options.enable_page_cache;
```
And here, it may be necessary to consider the default value:
```cpp
return _query_options.__isset.enable_page_cache ? 
            _query_options.enable_page_cache : !config::disable_storage_page_cache;
```